hrjobcontract/tests/phpunit/HRJobContractTestTrait.php

```diff
@@ -44,6 +44,55 @@ protected function createJobContract($contactID, $startDate = null, $endDate = n
   }
 
   /**
+   * Creates a new Job Contract for the given contact
+   *
+   * This is a little bit different than createJobContract
+   * its needed to test getCurrentContract
+   * TODO: change createJobContract behavior and remove this
+   * @param array $params should contain contact_id, start_date, end_date, position, title
+   * @throws \CiviCRM_API3_Exception
+   */
+  protected function createContract($params) {
```

Comment:
This class already has a method to create a contract (createJobContract()), I think both method should be merged